common/lib/xmodule/xmodule/css/sequence/display.scss

```diff
@@ -45,11 +47,12 @@ nav.sequence-nav {
     @include linear-gradient(left, $shadow, $transparent);
     position: absolute;
     top: 0;
-    left: 0;
+    left: ($baseline*2);
     width: 20px;
-    height: 46px;
+    height: 44px;
     background-color: transparent;
     pointer-events: none;
+    z-index: 40;
```

Comment:
If you're going to introduce a z-index here, can we at least manually use the LMS's Sass scale (https://github.com/edx/edx-platform/blob/master/lms%2Fstatic%2Fsass%2Fbase%2F_mixins.scss#L93)?